src/prebid.js

```diff
@@ -780,6 +780,7 @@ $$PREBID_GLOBAL$$.getConfig = config.getConfig;
  * ```
  */
 $$PREBID_GLOBAL$$.setConfig = config.setConfig;
+$$PREBID_GLOBAL$$.setBidderConfig = config.setBidderConfig;
```

Comment:
what was the thought process of making this a separate method vs using another argument on setConfig?

Reply:
No major reason I think. I thought being a little more explicit was nice, but I could be happy with just using another argument as well.